tests/MailsMockTraitTest.php

```diff
@@ -118,6 +118,20 @@ public function testMailWithExport()
         $this->assertFileExists($this->getFixturePath('test_mail_with_export.html'));
     }
 
+    public function testMailWithGlobalExportMode()
+    {
+        putenv('FAIL_EXPORT_JSON=false');
+        $this->globalExportMode = true;
+
+        Mail::to('test@mail.com')->queue(new TestMail(['name' => 'John Smith']));
+
+        $this->assertMailEquals(TestMail::class, [
+            $this->mockedMail('test@mail.com', 'test_mail_with_global_export.html', 'Test Subject'),
+        ]);
+
+        $this->assertFileExists($this->getFixturePath('test_mail_with_global_export.html'));
```

Comment:
let's assert that exported file content equals to expected, but use the separate fixture for it